youtube_dl/aes.py

```diff
@@ -60,6 +60,32 @@ def aes_cbc_decrypt(data, key, iv):
     return decrypted_data
 
 
+def aes_cfb_decrypt(data, key, iv):
```

Comment:
Could you add a test into [test/test_aes.py](https://github.com/rg3/youtube-dl/blob/master/test/test_aes.py)? So that we  detect breakage better.

(Also if possible add a way to generate the encrypted data with openssl at [devscripts/generate_aes_testdata.py](https://github.com/rg3/youtube-dl/blob/master/devscripts/generate_aes_testdata.py)